modules/manage/pages/manage-logging/manage-logging.adoc

```diff
@@ -444,3 +446,54 @@ The Logging REST API provides the endpoints for retrieving log and diagnostic in
 
 To retrieve log information use the `/diag` and `/sasl_logs`
 xref:rest-api:logs-rest-api.adoc[REST endpoints].
+
+[#adjust-threshold-slow-op-logging]
+== Adjusting Threshold for Logging Slow Operations
+
+It is possible to examine and/or alter the logging threshold for slow-running operations.
+This is done using the `mcctl` command that comes packaged with the Couchbase server installation.
+
+=== Getting Threshold Details
+The current settings are retrieved by using the `mcctl` cli to execute the `get sla` command:
+
+.Getting threshold details
+[source, bash]
+----
+/opt/couchbase/bin/mcctl --host localhost -u Administrator -P password \
+get sla
+----
+
+.Result
+[source, json]
+----
+{"comment":"Current MCBP SLA configuration",
+"version":1,
+"default":{"slow":"500 ms"}},
+"COMPACT_DB":{"slow":"1800 s"},
+"DELETE_BUCKET":{"slow":"10 s"},
+"SEQNO_PERSISTENCE":{"slow":"30 s"}
+}
+----
+
+The JSON message returned gives details of the operation being logged and the threshold time that will cause a timing message to be logged.
+
+=== Setting the Threshold
+
+The `mcctl` command line interface is also used to set the thresholds for the memcahe operations:
+
+.Set logging threshold example
+[source, bash]
+----
+/opt/couchbase/bin/mcctl --host localhost -u Administrator -P password \
+set sla '{"version":1, "DELETE_BUCKET":{"slow":"100ms"}}'
+----
+
+In this example, the threshold for the `DELETE_BUCKET` operation is being set to 100ms. If a bucket deletion operation takes longer than this, then an message will be logged. 
+It is also possible to set the threshold for all the op-codes in a single command by using the `default` code.
```

Comment:
What are the valid units and ranges?  `s` and `ms` are demonstrated.  can I set "999 us"?  can I set "1 ms"?

Reply:
wowza, nanos are rather pointless, but okay.  :)